### ISSUE TITLE
Regarding Class Code/Scripts

### ISSUE THREAD

**Issue by kaneclev** [kevinaangstadt/CS340-S23-Notes#102]
Hi all,

Just a quick note about code we produce in class and how we implement it into our study notes.
Given the fact that those who are reviewing the code in the notes wouldn't have the privilege of asking questions in class, I think it is a good idea for us to go over the code we are putting into our notes and make sure it is easily understood through comments in the code, sometimes even beyond what those we put in during class if need be.

If anyone has anything to add to this, suggestions are welcome.

**Comment by jaxlub** [comment]
I agree, I think formatting and commenting the code is important for comprehension for those that miss class. That being said much of what we have done is somewhat complex and has large conceptual background info so in addition to comments, we could also point them in the direction of that classes lecture recording.
ex.)
 <code from class w/ comments> 
See lecture 26 from <date> from ~15 mins to 25 mins.
<Link to class lecture>
